packages/next/src/server/app-render/get-layer-assets.tsx

```diff
@@ -4,6 +4,13 @@ import { getPreloadableFonts } from './get-preloadable-fonts'
 import type { AppRenderContext } from './app-render'
 import { getAssetQueryString } from './get-asset-query-string'
 
+function encodeFile(file: string) {
```

Comment:
do we need this repeated 3 times across different places? feels like this could get out of sync if someone goes and changes it further